### PR TITLE
[IMP] payment: update SIX screenshots

### PR DESCRIPTION
**Pull request by mivu-odoo**
If merged, this will update the screenshot for the "Connect a SIX Payment Terminal to your PoS" v14 user doc. In Odoo v14.0, the "Use a Payment Terminal" field only has "Six" as an option, not "Six without IoT Box." The new screenshot reflects this change and I also rewrote the step to reflect this change.

I also added alt text to the images, fixed the image file names, and fixed the image file folder name.

Related issue: https://github.com/odoo/documentation/issues/954

### CODE REVIEW

File: content/applications/sales/point_of_sale/payment/six.rst [hunk 1]
```diff
@@ -1,54 +1,55 @@
...
 Configuration
 =============
 
 Configure the Payment Method
 ----------------------------
 
-First, make sure that the POS Six module is installed. For this, go to *Apps*,
-remove the "Apps" filter and search for "POS Six".
+First, make sure that the :guilabel:`POS Six`` module is installed. For this, go to
+:guilabel:`Apps`, remove the *Apps* filter, and search for *POS Six*.
```
Comment (StraubCreative) [inline content/applications/sales/point_of_sale/payment/six.rst:20]: Add `:guilabel:` for clickable UI filter.

```suggestion
:guilabel:`Apps`, remove the :guilabel:'Apps' filter, and search for *POS Six*.
```

File: content/applications/sales/point_of_sale/payment/six.rst [hunk 2]
```diff
@@ -1,54 +1,55 @@
...
 
 Configuration
 =============
 
 Configure the Payment Method
 ----------------------------
 
-First, make sure that the POS Six module is installed. For this, go to *Apps*,
-remove the "Apps" filter and search for "POS Six".
+First, make sure that the :guilabel:`POS Six`` module is installed. For this, go to
```
Comment (StraubCreative) [inline content/applications/sales/point_of_sale/payment/six.rst:19]: Remove extra backtick after `POS Six`
```suggestion
First, make sure that the :guilabel:`POS Six` module is installed. For this, go to
```

File: content/applications/sales/point_of_sale/payment/six.rst [hunk 3]
```diff
@@ -1,54 +1,55 @@
...
-Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`,
-Create a new payment method for SIX, select the payment terminal option
-"SIX without IoT Box", and enter your payment terminal IP address.
+Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`, create a new payment
+method for SIX, select the payment terminal option :guilabel:`SIX`, and enter the payment terminal
+IP address.
 
-.. image:: media/six_02.png
-    :align: center
+.. image:: six/new-payment-method-six.png
```
Comment (StraubCreative) [inline content/applications/sales/point_of_sale/payment/six.rst:30]: The file size is fine but you'll probably get dinged for 4k resolution during technical review.
Would reduce file width by about 50% if you can to around tablet breakpoint of ~787px (as opposed to 1500+ pixels"

**Review by StraubCreative — CHANGES_REQUESTED** [review]
@mivu-odoo 

The changes you made here were great 🎉 

I like that you cleaned up the lines that were breaking too early before the 100th char along with the wording/flow edits you made which were spot on. Nice job renaming the image files too so they were more contextual to what the images communicate.

Just a couple small revisions below:
• remove extra backtick (`) in line 19
• add another `:guilabel:` on line 20
• would reduce resolution of the image you added by about 50%

**Review by LoredanaLrpz — APPROVED** [review]
It seems fine :).
 
Thank you for this PR and the review @mivu-odoo and @StraubCreative. However, if any changes are needed regarding the pos pages, create a task so the person in charge will do it. 
If you're wondering who's in charge of what app, please refer to this [doc](https://docs.google.com/spreadsheets/d/18QSSwvAVjla5WiAPr78Jdtr3A8LD8s8K_n8NT4BAxAE/edit#gid=1664672763).

I wish you both a nice day,

**Comment by mivu-odoo** [comment]
> It seems fine :).
> 
> Thank you for this PR and the review @mivu-odoo and @StraubCreative. However, if any changes are needed regarding the pos pages, create a task so the person in charge will do it. If you're wondering who's in charge of what app, please refer to this [doc](https://docs.google.com/spreadsheets/d/18QSSwvAVjla5WiAPr78Jdtr3A8LD8s8K_n8NT4BAxAE/edit#gid=1664672763).
> 
> I wish you both a nice day,

Hi @LoredanaLrpz ! 

I'm sorry, it was my mistake. I'm in charge of IoT Box and I forgot that payment terminals were a category separate from IoT Box. Would you like to take over the PR from here, or do you want me to make the changes that @StraubCreative requested and close this PR?

**Comment by mivu-odoo** [comment]
Got permission from Loredana (lrpz) to continue with this PR, despite it not being in my app scope (thank you!)

**Review by mivu-odoo — COMMENTED** [review]
Hi @StraubCreative !

Thank you for the review. I made the following changes:

- Line 19: Deleted the extra backtick after `POS Six`
- Line 20: Added a `:guilabel:` to `Apps` filter
- Line 30: Updated the screenshot for `new-payment-method.png` so that it's smaller

Please let me know if there's anything else that needs work. Thank you :-)

**Review by StraubCreative — APPROVED** [review]
Looks good to me!
On to DR :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+ squash

**Pull request closed without merging**